src/meltano/cli/utils.py

```diff
@@ -58,8 +58,7 @@ def print(self) -> None:
         if self.printed:
             return
 
-        logger.error(str(self), exc_info=self)
-        click.secho(str(self), fg="red", err=True)
```

Comment:
This is the bit that's not too log-friendly, specially when logs are formatted as JSON.